web/src/stream_create.ts

```diff
@@ -363,11 +365,10 @@ function create_stream(): void {
     const principals = JSON.stringify(user_ids);
     set_current_user_subscribed_to_created_stream(user_ids.includes(current_user.user_id));
 
-    assert(stream_settings_components.new_stream_can_remove_subscribers_group_widget !== null);
-    const widget_value =
-        stream_settings_components.new_stream_can_remove_subscribers_group_widget.value();
-    assert(typeof widget_value === "number");
-    const can_remove_subscribers_group_id = widget_value;
+    assert(can_remove_subscribers_group_widget !== undefined);
+    const can_remove_subscribers_group = settings_components.get_group_setting_widget_value(
```

Comment:
Should this be called `can_remove_subscribers_group_value` to make the type as not a group object obvious?

Reply:
Updated.